modular_skyrat/modules/company_imports/code/objects/nri_surplus/ak25.dm

```diff
@@ -21,10 +21,12 @@
 	load_empty_sound = 'modular_skyrat/modules/gunsgalore/sound/guns/interact/ltrifle_magin.ogg'
 	eject_sound = 'modular_skyrat/modules/gunsgalore/sound/guns/interact/ltrifle_magout.ogg'
 	alt_icons = TRUE
-	company_flag = COMPANY_IZHEVSK
 	spread = 29
 	recoil = 0.1
 
+/obj/item/gun/ballistic/automatic/ak25/give_manufacturer_examine()
+	AddComponent(/datum/component/manufacturer_examine, COMPANY_IZHEVSK)
```

Comment:
Could be an element instead

Reply:
It could, theoretically speaking, be an element, however.

Reply:
1100 line long sec_haul gun file.